Adding some guidance on how to get the huawei-sun2000-dongle-powersensor configured correctly

natsu-chan opened the pull request:
adding information about how to activate modus/tcp within the inverter, since the default does not allow to connect via modes/tcp.

natsu-chan committed:
Adding some guidance on how to get the huawei-sun2000-dongle-powersensor configured correctly

adding information about how to activate modus/tcp within the inverter, since the default does not allow to connect via modes/tcp.

andig commented on templates/definition/meter/huawei-sun2000-dongle-powersensor.yaml:
```diff
@@ -4,6 +4,14 @@ products:
+requirements:
+  description:
+    de: |
+      Es kann notwendig sein, dass der Installateur mit Hilfe eines "Errichterzugangs" zunächst "Modbus/TCP" in den Kommunikationseinstellungen des Wechselrichters freischaltet. 
```
Ich fände eine knappe Formulierung besser: 

    Erfordert ModbusTCP. Siehe ...

natsu-chan replied:
Ich bin auch immer für knappe Formulierungen, aber ich hätte mit "Erfordert MobusTCP" nichts anfangen können :D Dann wäre ich nicht in der Lage gewesen meinem Installateur zu sagen, was ich haben will.... 
Und der Link könnte potentiell in Zukunft mal kaputt gehen, wenn Huawei die Doku umzieht...

andig replied:
Erfordert ModbusTCP. Freischaltung über den "Errichterzugang". Siehe ...

natsu-chan replied:
passe ich an :)

natsu-chan committed:
shortening description

andig commented:
Do we need the same comment in the non-powersensor tempalte?

natsu-chan commented:
> Do we need the same comment in the non-powersensor tempalte?

I'm not sure. Might be the case. 
I don't know whether this configuration takes place in the dongle or in the inverter itself. 
Maybe one could add it with a "might be the case" in front of it? And hope somebody in the future gives feedback whether it is really necessary?

andig commented:
Dongle is used in both tempaltes!

premultiply commented:
/cc @RTTTC

RTTTC commented:
> Do we need the same comment in the non-powersensor tempalte?

Yes we do.

RTTTC commented:
While we are fixing these templates - one template has:
 `energy` `decode: uint32nan` and the other one has `decode: uint32`
`power`  `int32nan` and `int32`
Which one is right / better (nan vs no nan) ( and why?)

PS - do we use `energy` in `PV meter` at all ?? I thought - only `power`

natsu-chan commented:
> > Do we need the same comment in the non-powersensor tempalte?
> 
> Yes we do.

I will add that.

natsu-chan commented:
> While we are fixing these templates - one template has: `energy` `decode: uint32nan` and the other one has `decode: uint32` `power` `int32nan` and `int32` Which one is right / better (nan vs no nan) ( and why?)
> 
> PS - do we use `energy` in `PV meter` at all ?? I thought - only `power`

I have no idea, could you tackle that topic seperately? Actually has nothing todo with my topic.

natsu-chan committed:
Adding some guidance on how to get the huawei-sun2000-dongle configured correctly

adding information about how to activate modus/tcp within the inverter, since the default does not allow to connect via modes/tcp.
